recipes/drflac/all/conanfile.py

```diff
@@ -0,0 +1,82 @@
+from conans import ConanFile, CMake, tools
+
+required_conan_version = ">=1.33.0"
+
+
+class DrflacConan(ConanFile):
+    name = "drflac"
+    description = "FLAC audio decoder."
+    homepage = "https://mackron.github.io/dr_flac"
+    topics = ("audio", "flac", "sound")
+    license = ("Unlicense", "MIT-0")
+    url = "https://github.com/conan-io/conan-center-index"
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False], 
+        "fPIC": [True, False],
+        "buffer_size": "ANY",
```

Comment:
I first tried
```python
"buffer_size": int,
```
here but it does not work. If there is any way to tell Conan that this option only accepts integer values, let me know.

Reply:
No, but because the python used on Conan converts everything to string, we can't handle it. Another option, is adding a list with possible value, but as buffer size is too flexible, ANY is the best choice.